Header header

object_tracker/RotationParameters rotation_parameters

object_recognition_msgs/RecognizedObjectArray objects
float64[] radius
float64[] phase